flow-typed/npm/downshift_v2.x.x.js.flow

```diff
@@ -209,7 +209,7 @@ declare module downshift {
     openMenu: (cb?: Callback) => void;
     closeMenu: (cb?: Callback) => void;
     toggleMenu: (otherStateToSet?: {}, cb?: Callback) => void;
-    selectItem: (item: Item, otherStateToSet?: {}, cb?: Callback) => void;
+    selectItem: (item: ?Item, otherStateToSet?: {}, cb?: Callback) => void;
```

Comment:
i never used flow. is this correct syntax? i have copied it from here: https://flow.org/en/docs/types/maybe/

but maybe (hah), rather `null` should be explicitly stated instead of a maybe type to disallow `undefined`?

Reply:
ok i changed it to `| null` and also edited the state type as well to include `null` for `selectedItem`, that was missing from the flow types, it was there for the typescript types already